test/service.thrift

```diff
@@ -0,0 +1,7 @@
+
+service Foo {
+    byte foo(0: byte number) throws (
```

Comment:
@abhinav Can you check? This is the args struct, not the result struct. 0 *is* reserved on the result and disallowed on the throws side. I can add a test to verify.